boards/DIM/Inc/App/App_RegenPaddle.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <stdint.h>
+#include "App_ErrorCode.h"
+
+/**
+ * Allocate and initialize a regen paddle
+ * @param get_paddle_position A function that can be called to get the paddle
+ *                            position as a percentage, where 0% means not
+ *                            pressed at all and 100% means pressed all the way
```

Comment:
could we change `0%` and `100%` to just `0` and `100` here? IMO it's potentially ambiguous what number `100%` maps to here.

Reply:
Ah yeah that's true, fixed